apps/frontpage/scripts/copy-other-sitemaps.ts

```diff
@@ -34,11 +34,11 @@ const OTHER_SITEMAPS = {
 const DESTINATION = path.join(__dirname, '../public/sitemap');
 const SITEMAP_FILENAME = 'sitemap.xml';
 
-function stripDirname(file: string) {
-  return file.replace(/.*(\/public\/.*)/, '$1');
+function stripDirname(file: string): string {
+  return file.replace(/.*(?<temp1>\/public\/.*)/, '$1');
```

Comment:
What is `<temp1>` here?

Reply:
This is what was recommended by ESLint.
> Capture group '(\/public\/.*)' should be converted to a named or non-capturing group.
https://eslint.org/docs/latest/rules/prefer-named-capture-group

Reply:
Oh, I completely forgot about that feature. Thanks.